pytorch_pretrained_bert/convert_tf_checkpoint_to_pytorch.py

```diff
@@ -60,7 +67,15 @@ def convert_tf_checkpoint_to_pytorch(tf_checkpoint_path, bert_config_file, pytor
                         type = str,
                         required = True,
                         help = "Path to the output PyTorch model.")
+    ## Optional parameters
+    parser.add_argument("--bert_model",
+                        default = None,
+                        type = str,
+                        required = False,
+                        help = "Type of the model which is going to be converted. \n"
+                               "This specifies the final layer.")
     args = parser.parse_args()
     convert_tf_checkpoint_to_pytorch(args.tf_checkpoint_path,
                                      args.bert_config_file,
-                                     args.pytorch_dump_path)
+                                     args.pytorch_dump_path,
+                                     args.bert_model)
```

Comment:
Maybe it would be better to add a dictionary of variable conversions here that would be, in this case `{'squad': 'qa_output'}.

This way we can accomodate other type of models like in #580.

This PR should let us also do the workflow of #580 without problems.

Reply:
Makes sense, we can add this to the config mentioned in the [discussion above](https://github.com/huggingface/pytorch-pretrained-BERT/pull/521#discussion_r282359014).